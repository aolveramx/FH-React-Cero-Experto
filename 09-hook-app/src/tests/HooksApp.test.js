import {shallow} from 'enzyme'
import HooksApp from "../HooksApp";


describe('Tests in <HookApp />', () => {

  
  test('should return snapshoot of <HookApp />', () => {
    
    const wrapper = shallow(<HooksApp />)
    
    expect(wrapper).toMatchSnapshot()

  });
});