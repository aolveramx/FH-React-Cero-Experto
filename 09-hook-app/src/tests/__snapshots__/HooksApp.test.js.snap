// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <HookApp /> should return snapshoot of <HookApp /> 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
